.github/workflows/twister.yaml

```diff
@@ -292,10 +292,14 @@ jobs:
         name: Upload to opensearch
         run: |
           pip3 install elasticsearch
+          # set run date on upload to get consistent and unified data across the matrix.
+          run_date=`date --iso-8601=minutes`
```

Comment:
I just wonder if getting upload date stored as run date won't cause confusion by users at some point. E.g. scheduled runs take quit some time to finish. I can imagine looking at the report, seeing run date and assuming, that it corresponded to the state of main at that time, even though main changed multiple times. I know there is also commit id, commit date etc. so it's just a minor thing to consider.

Reply:
Commit message says:

> To get consistent and 1 single rundate per CI run, set run_date when
we upload the data. This is optional mostly for CI and to avoid
duplication when we re-run jobs.

Reply:
yes, saw this, but still, at least for me, run date and upload date doesn't mean the same. Even if it is mostly for CI, we store this date. IMO if at any point someone will try to get any correlation with run date  they would get a correlation with upload date. I believe the same consistency between jobs can be achieved when the date is read and store as gh env at the beginning of the workflow and used the same way during updating.